### DISCUSSION TITLE
how will i source my image so i can upload it along with my html if i want to host my website on github

### DISCUSSION THREAD

**Discussion by Najeeb-Ayo** [community/community#146436] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

i have finish writing my code for the website and the overview is as i wanted but when i host it on github the image i added to my code does not display with it, how do i sources the image in my code so it can go live with the site, i still have it on my system though

**Comment by cachedcake** [comment, marked as answer]
Hello again, @Najeeb-Ayo 

I've went through your repository and I've spotted your problem as you're linking the image wrong. Try this:

```html
<img src="https://github.com/Najeeb-Ayo/challenge/blob/main/image-qr-code.png?raw=true" alt="the image of the qr code to be scan">
```
What I've Changed:
- Instead of directing it like a code file I copied the image address


> For any further questions you may have use the [GitHub Pages Documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#about-github-pages)

Let me know if this helps!

**Reply by mecodeatlas** [reply]
Hey @Najeeb-Ayo! Did @cakecable's response help you with your question?

**Reply by Najeeb-Ayo** [reply]
thank you for your reply
